nssp_demo/score_forecast.R

```diff
@@ -0,0 +1,209 @@
+suppressPackageStartupMessages(library(dplyr))
+suppressPackageStartupMessages(library(scoringutils))
+suppressPackageStartupMessages(library(arrow))
+suppressPackageStartupMessages(library(argparser))
```

Comment:
prefer the approach in `nssp_demo/postprocess_state_forecast.R`

Reply:
Done in 6e1148f